book/src/tooling/defmt.md

```diff
@@ -5,12 +5,13 @@ Ariel OS supports [defmt] on all platforms. It is enabled by default.
 See the [defmt documentation] for general info on `defmt`.
 
 In Ariel OS, the log level defaults to `info`. It can be configured using the
-laze variable `LOG`.
+laze variable `LOG`. The variable can be given multiple times for crate- or
+module-level granularity.
 
 Example:
 
 ```shell
-# laze build -C examples/log --builders nrf52840dk -DLOG=warn run
+# laze build -C examples/log --builders nrf52840dk -DLOG=warn -DLOG=ariel_os_rt=trace run
```

Comment:
```suggestion
# laze build -C examples/log --builders nrf52840dk -DLOG=warn -DLOG=ariel_os_rt=trace run
# laze build -C examples/log --builders nrf52840dk -DLOG=warn,ariel_os_rt=trace run
```

Comma-seperated is also possible. ... did I say, tested it, and it doesn't work. That's a laze bug, the `--define/-D` had `,` as value separator like builders/apps.

Reply:
https://github.com/kaspar030/laze/pull/590